langchain/vectorstores/opensearch_vector_search.py

```diff
@@ -359,6 +362,7 @@ def add_texts(
             embeddings,
             texts,
             metadatas,
+            ids,
```

Comment:
we should probably be passing these as kwargs, i can update

Reply:
thanks!